packages/babel-plugin-proposal-pipeline-operator/src/index.js

```diff
@@ -1,18 +1,26 @@
 import { declare } from "@babel/helper-plugin-utils";
 import syntaxPipelineOperator from "@babel/plugin-syntax-pipeline-operator";
 import minimalVisitor from "./minimalVisitor";
-import smartVisitor from "./smartVisitor";
+import hackVisitor from "./hackVisitor";
 import fsharpVisitor from "./fsharpVisitor";
+import smartVisitor from "./smartVisitor";
 
 const visitorsPerProposal = {
   minimal: minimalVisitor,
-  smart: smartVisitor,
+  hack: hackVisitor,
   fsharp: fsharpVisitor,
+  smart: smartVisitor,
 };
 
 export default declare((api, options) => {
   api.assertVersion(7);
 
+  if (options.proposal === "smart") {
```

Comment:
Not really part of this PR, but we throw an error if `!visitorsPerProposal.hasOwnProperty(options.proposl)`.

Reply:
Configuring the plugin without a `proposal` already throws an error: https://github.com/js-choi/babel/blob/11a7a491b57c7c1e2b33cf99a9d244c6eb6e32d6/packages/babel-plugin-syntax-pipeline-operator/src/index.js#L11-L16.

Do we still want to throw an error specifically in `babel-plugin-proposal-pipeline-operator/src/index.js`, too?

I added a test case covering the no-proposal case in 518cf048a215af9f9a872129e325e0affbe86dd4.